### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by zer0x64**

### CODE REVIEW

**Comment by zer0x64** [comment]
CI's fixed, although more stuff could be added. For instance, there are no WASM tests yet and clippy warnings are not hard errors for now(since there are a bunch of warnings to fix), but that's still a lot better than no CI at all.

**Comment by CBenoit** [comment]
Agreed, this can be improved later :slightly_smiling_face:

**Review by CBenoit — APPROVED** [review]

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -16,11 +16,39 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+      with:
+        fls: true
```
Comment (junior-n30) [inline .github/workflows/rust.yml:20]: typo? should it be `lfs`?

Comment (zer0x64) [reply .github/workflows/rust.yml:20]: Ohhhh, that's explains why the LFS files weren't checked out

**Comment by CBenoit** [comment]
Now it's perfect

**Pull request merged**